src/student.rb

```diff
@@ -0,0 +1,10 @@
+class Student < Person
+  def initialize(age, classroom, name = 'Ahad', parent_permission: true)
```

Comment:
- Optional.
I would recommend defaulting the name to unknown instead of your own name since if the name is not provided, it will take your name as default and the user wouldn't know whether the name is unknown or provided.